kie-server-parent/kie-server-controller/kie-server-controller-standalone/src/main/java/org/kie/server/controller/common/StandaloneControllerApplication.java

```diff
@@ -38,6 +43,10 @@ public StandaloneControllerApplication() {
                 add(new StandaloneKieServerControllerImpl());
                 add(new StandaloneSpecManagementServiceImpl());
                 add(new StandaloneRuntimeManagementServiceImpl());
+                if(SWAGGER_DISABLED == false) {
```

Comment:
`SWAGGER_DISABLED` is always non-null so I think we can make use of `!SWAGGER_DISABLED`.

Reply:
@MarianMacik true, that is just my personal coding style, I believe it is easier to read the code this way.